assets/templateFloxEnv/manifest.toml

```diff
@@ -9,20 +9,32 @@
 
 [install]
 _FLOX_INIT_INSTALL
-# Set an environment variable.
-# These variables may not reference once another
-# and are added to the environment without expansion.
+# Set environment variables in the 'vars' section.
```

Comment:
(comment not relevant to this line)
nit: the PR description says hook.script is deprecated, but was it deprecated or we just added a deprecation notice to the man page?

Reply:
It's deprecated in the sense that we no longer add it by default to the manifest template, we discourage users from adding new `script`s, and in the next PR we'll start displaying a warning.